test/testlib/minikube_utilities.go

```diff
@@ -544,13 +544,27 @@ func GetDatabaseIncarnation(t *testing.T, namespace string, podName string, data
 	return nil
 }
 
+func GetRestartCount(t *testing.T, podName string, options *k8s.KubectlOptions) int32 {
+	pod := k8s.GetPod(t, options, podName)
+
+	var restartCount int32
+	for _, status := range pod.Status.ContainerStatuses {
+		restartCount += status.RestartCount
+	}
+
+	return restartCount
+}
+
+// TODO remove databaseNae param - and retrieve it from extracted databaseOptions
 func AwaitDatabaseRestart(t *testing.T, namespace string, podName string, databaseName string, databaseOptions *helm.Options, restart func()) {
 	incarnation := GetDatabaseIncarnation(t, namespace, podName, databaseName)
 
 	restart()
 
 	Await(t, func() bool {
-		return GetDatabaseIncarnation(t, namespace, podName, databaseName).Major > incarnation.Major
+		newIncarnation := GetDatabaseIncarnation(t, namespace, podName, databaseName)
+		t.Log("incarnation ->", newIncarnation)
```

Comment:
do you really need this?

```
TestKubernetesRestoreDatabase 2020-11-06T06:32:38Z logger.go:66: Running command kubectl with args [--namespace testkubernetesrestoredatabase-nrbqi3 exec admin-nrbqi3-nuodb-cluster0-0 -- nuocmd --show-json get databases]
TestKubernetesRestoreDatabase 2020-11-06T06:32:41Z logger.go:66: {
TestKubernetesRestoreDatabase 2020-11-06T06:32:41Z logger.go:66:   "incarnation": {
TestKubernetesRestoreDatabase 2020-11-06T06:32:41Z logger.go:66:     "major": 2, 
TestKubernetesRestoreDatabase 2020-11-06T06:32:41Z logger.go:66:     "minor": 0
TestKubernetesRestoreDatabase 2020-11-06T06:32:41Z logger.go:66:   }, 
TestKubernetesRestoreDatabase 2020-11-06T06:32:41Z logger.go:66:   "name": "demo", 
TestKubernetesRestoreDatabase 2020-11-06T06:32:41Z logger.go:66:   "state": "AWAITING_ARCHIVE_HISTORIES_MSG", 
TestKubernetesRestoreDatabase 2020-11-06T06:32:41Z logger.go:66:   "uri": "https://localhost:8888/api/1/databases/demo"
TestKubernetesRestoreDatabase 2020-11-06T06:32:41Z logger.go:66: }
    minikube_utilities.go:566: incarnation -> &{2 0}
```

Reply:
I was getting a timeout on the incarnation number changing - so I added logging to understand the value at each attempt, rather than just the final value.

I will be removing this line, now that the test is passing reliably.